packages/protocol/contracts/L1/TaikoL1.sol

```diff
@@ -276,11 +279,11 @@ contract TaikoL1 is EssentialContract, IHeaderSync, TaikoEvents {
         return LibAnchorSignature.signTransaction(hash, k);
     }
 
-    function getBlockProvers(
+    function getForkChoice(
         uint256 id,
         bytes32 parentHash
-    ) public view returns (address[] memory) {
-        return state.forkChoices[id][parentHash].provers;
+    ) public view returns (TaikoData.ForkChoice memory) {
```

Comment:
changed this method to return entire fork choice so we can verify integrity in testing.

Reply:
not sure but if these method extensions / overrides are not necessary in the real world use case, maybe we can move them into `./contracts/test/L1/TestTaikoL1.sol`?

Reply:
Agreed that makes sense!